### PR TITLE
Add ability to display movie title and picture

### PR DESCRIPTION
**Pull request by geet084**
- it didn't look like we needed the constructor yet/at all...?
- figured out the link for displaying the pictures and built out the initial movie "card"

### CODE REVIEW

File: src/containers/App/App.js [hunk 1]
```diff
@@ -5,30 +5,40 @@ import { connect } from 'react-redux';
 import { getMovies } from '../../actions';
 import API from '../../utils/api';
 import { apiKey } from '../../utils/api-key';
+import Movie from '../Movie/Movie'
 
 class App extends Component {
-  constructor(props) {
-    super(props);
-
-  }
```
Comment (kaohman) [inline src/containers/App/App.js]: Agree we don't need to constructor, at least not yet. I ran into the same thing building out ToDo Box where app is stateless.

File: src/containers/App/App.js [hunk 2]
```diff
@@ -5,30 +5,40 @@ import { connect } from 'react-redux';
...
 
   componentDidMount = async () => {
-    const url = `https://api.themoviedb.org/3/movie/now_playing?api_key=${apiKey}&language=en-US&page=1`;
+    const initialCategory = 'popular'
+    const root = `https://api.themoviedb.org/3/movie/${initialCategory}`
+    const url = `${root}?page=1&api_key=${apiKey}&language=en-US`
```
Comment (kaohman) [inline src/containers/App/App.js]: I like how you made this dynamic if we want to change where we are pulling from in the future

File: src/containers/App/App.js (hunk 1 above)
Comment (geet084) [reply src/containers/App/App.js]: I'm hoping that we may not need it at all, it feels a little unnecessary to be honest... but we'll see

File: src/containers/App/App.js (hunk 2 above)
Comment (geet084) [reply src/containers/App/App.js]: I think the initial category will turn into current category and be routed accordingly, but it's a place to start at least

**Pull request merged**